src/cdk/table/table.scss

```diff
@@ -1,3 +1,15 @@
 .cdk-table-fixed-layout {
   table-layout: fixed;
 }
+
+.cdk-table-virtual-scroll {
+  .cdk-header-row.cdk-table-sticky,
+  .cdk-header-cell.cdk-table-sticky {
+    will-change: top;
```

Comment:
Are we sure we need this? If so can you please document why it is necessary. `will-change` is a notorious foot-gun https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

Reply:
After more testing, this didn't yield better performance. I'll revert this change.